packages/openapi-fetch/test/index.test.ts

```diff
@@ -1275,6 +1277,186 @@ describe("client", () => {
     });
   });
 
+  describe("body serialization", () => {
+    const BODY_ACCEPTING_METHODS = [["PUT"], ["POST"], ["DELETE"], ["OPTIONS"], ["PATCH"]] as const;
+    const ALL_METHODS = [...BODY_ACCEPTING_METHODS, ["GET"], ["HEAD"]] as const;
+
+    const fireRequestAndGetBodyInformation = async (options: {
+      bodySerializer?: BodySerializer<unknown>;
+      method: (typeof ALL_METHODS)[number][number];
+      fetchOptions: FetchOptions<any>;
+    }) => {
+      const client = createClient<any>({ baseUrl, bodySerializer: options.bodySerializer });
+      const { getRequest } = useMockRequestHandler({
```

Comment:
I assume this is piling one mock over the other. I didn't find any "cleanup" procedure in the existing tests, so piling several more mocks on top 🙂

Reply:
All good, the cleanup is right here! https://github.com/openapi-ts/openapi-typescript/blob/449d7e86369e8aa7788571ae67e00e6869e0d06f/packages/openapi-fetch/test/index.test.ts#L21

Reply:
Ha, missed that. Thanks 👍